src/bindgen_utils.rs

```diff
@@ -14,8 +14,7 @@ use std::io;
 use std::path::Path;
 use walkdir::WalkDir;
 
-/// Recursively copy all files with the given extension from the source to the
-/// targt directories.
+/// Recursively copy all files with the given extension from the source to the targt directories.
```

Comment:
there was also a typo in the comment, I think:
```suggestion
/// Recursively copy all files with the given extension from the source to the target directories.
```